### PR TITLE
Fix scheduled

### PR DESCRIPTION
**Pull request by pawelru**
fix verdepcheck tests
rhub continue to fail - I opened a separate issue in rhub org to solve it

### CODE REVIEW

File: .github/workflows/scheduled.yaml [hunk 1]
```diff
@@ -48,19 +48,20 @@ jobs:
...
+  #   uses: insightsengineering/r.pkg.template/.github/workflows/revdepcheck.yaml@main
   rhub:
     if: >
       github.event_name == 'schedule' || (
         github.event_name == 'workflow_dispatch' &&
         inputs.chosen-workflow == 'rhub'
       )
     name: R-hub 🌐
-    uses: insightsengineering/r.pkg.template/.github/workflows/rhub.yaml@main
+    uses: insightsengineering/r.pkg.template/.github/workflows/rhub.yaml@rhub_skip_quarto
```
Comment (pawelru) [inline .github/workflows/scheduled.yaml:67]: that needs to be reverted

Comment (pawelru) [inline .github/workflows/scheduled.yaml:67]: revert after merge of https://github.com/insightsengineering/r.pkg.template/pull/258

```suggestion
    uses: insightsengineering/r.pkg.template/.github/workflows/rhub.yaml@main
```

**Comment by pawelru** [comment]
That has just been merged and released. Let me trigger another run now

**Comment by averissimo** [comment]
I noticed that and I rerun the last run. The problem seems fixed, although there are some platforms that fail

https://github.com/insightsengineering/verdepcheck/actions/runs/11290686165/job/31412439902

image

**Comment by pawelru** [comment]
Yeah. Let me do that next week. Converting this to draft.

**Comment by pawelru** [comment]
all green here: https://github.com/insightsengineering/verdepcheck/actions/runs/11325617876

**Review by m7pr — APPROVED** [review]

**Pull request merged**